app/views/landing_page/blocks/_image.html.erb

```diff
@@ -1,5 +1,10 @@
+<% 
+  img_class = ("govuk-!-width-full" if block.data["theme"] == "full_width")
+  img_class << " govuk-!-margin-bottom-6"
```

Comment:
Apparently these should be used for overriding only, but I think considering the time pressures here I'm okay with it, but we might circle back and redo later.

Reply:
Sounds good, thanks Andy 👍